docs/pages/guides/adopting-prebuild.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Adopting Prebuild
+---
+
+import { Terminal } from '~/ui/components/Snippet';
+
+There are many advantages of using Prebuild to manage your native projects. This guide will show you how to adopt prebuilding in a project that was bootstrapped with `npx react-native init`.
+
+> **Note:** Adopting prebuild automatically adds support for developing modules with the [Sweet API][expo-modules-core] by linking `expo-modules-core` natively. You can also use any command from Expo CLI `npx expo` in your project.
+
+## Bootstrap a new project
+
+<!-- NOTE: Update the version when we bump support -->
+
+<Terminal cmd={["$ npx react-native init --version 0.69.4"]} />
+
+This will create a new React Native project. Note that [not all versions of `react-native` are explicitly supported](/versions/latest/#each-expo-sdk-version-depends-on-a), be sure to use a version of `react-native` that has a corresponding Expo SDK version.
+
+## Install the `expo` package
+
+The `expo` package contains the `npx expo prebuild` command and indicates which prebuild template to use:
+
+<Terminal cmd={["$ yarn add expo"]} />
+
+Ensure you install the version of `expo` that works for your currently installed version of `react-native`. Note that not all major versions of `react-native` are supported.
+
+## Update the entry file
+
+Modify the entry file like so:
+
+```diff
++ import {registerRootComponent} from 'expo';
+
+- import {AppRegistry} from 'react-native';
+import App from './App';
+- import {name as appName} from './app.json';
+
+- AppRegistry.registerComponent(appName, () => App);
++ registerRootComponent(App);
+```
+
+Alternatively, you can change `appName` to `"main"`. Switching to `registerRootComponent` is useful if you want to take advantage of other features like web support. Importing `expo` adds side-effects for development environments that are removed in production bundles.
+
+## Prebuild
```

Comment:
Why would you manually want to run prebuild at this point? If the npm scripts are updated to call `expo run:...` below, then wouldn't it be much easier to just say: 'If you haven't made any native file modifications in your project, it's safe to delete the ios and android folders at this point. They will be generated automatically for you when you start your project`

Reply:
my take is that it's just more explicit to run prebuild here, and the app might not actually work yet if you were to go straight to `expo run:ios` or `expo run:android`